### PR TITLE
Fjern intransaction for tilgangtilperson som håndterer ds selv

### PR DESCRIPTION
**Pull request by sebassonav**
Prøver å løse concurrent serializble https://logs.adeo.no/app/r?l=DISCOVER_SINGLE_DOC_LOCATOR&v=8.12.2&lz=N4IglgdgJgpgHiAXCAnANhmgLADgMYAMAtCjgQIZECMVMKROAzBUTFQOwBGnU2nWUTiAA0IAE4B7AO4BJaPCQgANhIDmAZwAu5dQAsi5AA6H1RQ5KhECBNOkYjx0mVEUAFMJzgBhAJpYAQgScAK4AYqEo%2FowAagD6AHIAEg5iMABmMGKpYooA9EaGuVBg6ngSAG6ZAMS5APyxqgC8ABRpYEqameqIAITNAJTCmmAAtjCIrZIjiADkAEwEc1hWjFZYACpUBIhYc4gArGgAdATsVABaM0MSswtLK1Y463NUiIxYiKdHWCj7l%2F39ABksXILTKSmCIwg3T6ShglSUwjG6nU5FUMGEMAg5TwSh06mEBSUYDw5GGEggwi05DwAGtYpoxDSYIM2h0ur1ms0ZgASamdGYTLQSVKIAoAZW0nUGY20E3IxJ0vUpxVRnDhUF6aWEbRgSk1EHIyMMzOEkFgcFmXH2MDIeHY1Ht9AEBH2DAIOEscxwOHYWH2cxQNKwaSutJgAE9EIbjaaIDBVGTxj1tSamSNus0AI7BTJRmCaTpiPGFllDCOGcaGXRM9RlnN5iYjMl4XSxau18bNGMwdQmvDjAtFkvSgHCbl8qUwQXNYWiiVTmUF8jyxUwlUlcjqmCalM6sB6zVYnF4lFm%2BSWmbW20Ee2O9jOqCu92eoje33%2BwPB0PCcP57G4viwjxomnRasIaZGpmDZiFGhiDJoFZVjWOj1rmsFNi2bYdqhEzHoBKKIPBY4Tvy05CpoIrjAuSZLnKzQKmASo9BuaoauB5rwLM6CYLghAkGQlA0HQDDMJQbBcDwfACJwYaRmKxjEqS5KUiBSbgZBGYTDB%2BalsWZKdEQnAwLo5DQMSECqAhSFEShdaDDpmGaK27Z2V2RIkmSYAUogQ6ZCOMBGSZZlQBZVkAoMnGXjx2D4MQpAsMJ9BMCwEncLwWD8IIVyOc0eKWcEaLjBCA7xsIjkzNSdIMkyA6IAApHMcwiqoRyGBIWiqKk6hZkoRzBMMfWuOKACKAAyACicADoYKkAFSNXMMwAiAAC%2BQA%3D%3D%3D

### CODE REVIEW

File: apps/etterlatte-behandling/src/main/kotlin/behandling/tilgang/TilgangRoutes.kt [hunk 1]
```diff
@@ -36,12 +35,10 @@ internal fun Route.tilgangRoutes(tilgangService: TilgangService) {
             val fnr = call.receive<String>()
             val harTilgang =
                 harTilgangBrukertypeSjekk(brukerTokenInfo) { _ ->
-                    inTransaction {
-                        tilgangService.harTilgangTilPerson(
-                            fnr,
-                            Kontekst.get().appUserAsSaksbehandler().saksbehandlerMedRoller,
-                        )
-                    }
+                    tilgangService.harTilgangTilPerson(
```
Comment (sebassonav) [inline apps/etterlatte-behandling/src/main/kotlin/behandling/tilgang/TilgangRoutes.kt:38]: Denne bruker `ds.connection.use` og rydder opp etter seg selv direkte så `inTransaction` her er overflødig.

File: apps/etterlatte-behandling/src/main/kotlin/tilgangsstyring/Tilgangsstyring.kt [hunk 2]
```diff
@@ -138,12 +137,10 @@ suspend inline fun PipelineContext<*, ApplicationCall>.withFoedselsnummerInterna
     when (brukerTokenInfo) {
         is Saksbehandler -> {
             val harTilgang =
-                inTransaction {
-                    tilgangService.harTilgangTilPerson(
-                        foedselsnummer.value,
-                        Kontekst.get().appUserAsSaksbehandler().saksbehandlerMedRoller,
-                    )
-                }
+                tilgangService.harTilgangTilPerson(
```
Comment (sebassonav) [inline apps/etterlatte-behandling/src/main/kotlin/tilgangsstyring/Tilgangsstyring.kt:140]: Samme her

Comment (sebassonav) [reply apps/etterlatte-behandling/src/main/kotlin/tilgangsstyring/Tilgangsstyring.kt:140]: Denne er brukt som wrapper flere steder som kan være årsaken til at man får  ERROR: could not serialize access due to read/write dependencies among transactions

**Comment by sebassonav** [comment]
testet ok i dev

**Review by ingleivj — APPROVED** [review]

**Review by oyvindsh — APPROVED** [review]

**Pull request merged**